app/src/main/java/org/sagebionetworks/research/mpower/research/DataSourceManager.kt

```diff
@@ -109,17 +122,15 @@ data class StudyBurstConfiguration(
          * @property completionTasks for each day of the study burst.
          */
         val completionTasks: Set<CompletionTask> = setOf(
-                CompletionTask(linkedSetOf(STUDY_BURST_REMINDER, DEMOGRAPHICS), 1),
-                CompletionTask(linkedSetOf(BACKGROUND), 9),
-                CompletionTask(linkedSetOf(ENGAGEMENT), 14)),
+                // TODO: mdephillips 10/16/18 Add study burst reminder and engagement
+                // TODO: mdephillips 10/16/18 back in when development is done on those features
+                CompletionTask(linkedSetOf(/**STUDY_BURST_REMINDER, */DEMOGRAPHICS), 1),
```

Comment:
Uncomment these to enable them.  Engagement survey is ready to go and works, but reminders isn't ready and probably won't be built on ResearchStack.